spec/line/bot/send_message_05_location_spec.rb

```diff
@@ -30,6 +30,34 @@
     expect(response.body).to eq(expected)
   end
 
+  it 'pushes the location message with additional payload' do
+    uri_template = Addressable::Template.new Line::Bot::API::DEFAULT_ENDPOINT + '/bot/message/push'
+    stub_request(:post, uri_template).to_return { |request| {body: request.body, status: 200} }
+
+    client = Line::Bot::Client.new do |config|
+      config.channel_token = 'channel_token'
+    end
+
+    user_id = 'user_id'
+    message = {
+      'type' => 'location',
+      'title' => 'LINE Corporation.',
+      'address' => 'Hikarie  Shibuya-ku Tokyo 151-0002',
```

Comment:
This data is little old :D

Reply:
I'll change test data to more LINE-neutral one like National Diet Building.

Reply:
done